build-coordinator/src/test/java/org/jboss/pnc/core/builder/coordinator/bpm/BpmBuildSchedulerTest.java

```diff
@@ -48,6 +46,7 @@
 
 public class BpmBuildSchedulerTest {
 
+    @Ignore //TODO what exactly are we testing here?
```

Comment:
After the refactor how parameters are put together I don't think this test is relevant anymore. We have two set of parameters `processParameters` which is a map serialized to json. Testing means we are testing `ObjectMapper`.
And second is `BuildExecutionConfigurationREST` which serialization is tested else where.
The only thing remaining here is `BpmModuleConfig` which should also be tested elsewhere.
Do you think we need to test something else? 
Parameter preparation doesn't have anything with process starting, this test is too complex to test the parameters.
I would remove it, wdyt ?